lms/djangoapps/commerce/views.py

```diff
@@ -103,28 +103,26 @@ def post(self, request, *args, **kwargs):  # pylint: disable=unused-argument
 
         # Make the API call
         try:
-            order_number, order_status, _body = api.create_order(user, honor_mode.sku)
-            if order_status == OrderStatus.COMPLETE:
-                msg = Messages.ORDER_COMPLETED.format(order_number=order_number)
+            response_data = api.create_basket(
+                user,
+                honor_mode.sku,
+                payment_processor="cybersource",
+            )
+            if response_data['payment_data'] is not None:
```

Comment:
I'd assign the result of `response_data['payment_data']` to a variable, say `payment_data`, so you don't need to retrieve it twice. It'd also be a little more Pythonic if this was:
```python
if payment_data:
```